include/bluetooth/l2cap.h

```diff
@@ -414,6 +422,19 @@ int bt_l2cap_br_server_register(struct bt_l2cap_server *server);
 int bt_l2cap_ecred_chan_connect(struct bt_conn *conn,
 				struct bt_l2cap_chan **chans, uint16_t psm);
 
+/** @brief Reconfigure Enhanced Credit Based L2CAP channels
+ *
+ *  Reconfigure up to 5 L2CAP channels. Channels must be from the same bt_conn.
+ *  Once reconfiguration is completed each channel reconfigured() callback will
+ *  be called. MTU cannot be decreased on any of provided channels.
+
+ *  @param chans Array of channel objects
```

Comment:
We should clarify this doc. Add "Null-terminated. Elements after the first 5 are silently ignored."

Reply:
OK, I've updated comment